it/repositories/MongoJourneyAnswersRepositoryISpec.scala

```diff
@@ -16,123 +16,97 @@
 
 package repositories
 
-import config.AppConfig
+import models.common.JourneyName
+import models.common.JourneyStatus._
 import models.database.JourneyAnswers
-import org.mongodb.scala.model.Filters
-import org.scalatest.BeforeAndAfterEach
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
+import org.scalatest.{BeforeAndAfterEach, OptionValues}
 import org.scalatestplus.play.guice.GuiceOneAppPerSuite
-import play.api.Application
-import play.api.inject.bind
-import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.json.Json
 import play.api.test.Helpers.{await, defaultAwaitTimeout}
-import play.api.test.Injecting
 import support.MongoTestSupport
 import uk.gov.hmrc.mongo.test.MongoSupport
+import utils.BaseSpec._
 
-import java.time._
-import java.util.concurrent.TimeUnit
+import java.time.Duration
 import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
+import scala.concurrent.duration._
 
 class MongoJourneyAnswersRepositoryISpec
```

Comment:
Should we add more test cases for unhappy paths. E.g. when there is no stored state:

- Unable to retrieve it
- Unable to update the status

How about tests to check the presence of indexes (there were tests here before for that). Does this give us a safety net incase they're removed by accident?